include/swift/Option/Options.td

```diff
@@ -282,6 +282,12 @@ def no_static_stdlib: Flag<["-"], "no-static-stdlib">,
   Flags<[HelpHidden]>,
```

Comment:
How about removing that `static_stdlib` and `no_static_stdlib` flag? Your changes removes their functionality.